### PR TITLE
Onderyildirim added levelup

### PR DESCRIPTION
**Pull request by onderyildirim**

### CODE REVIEW

**Review by kisshetty — APPROVED** [review]
This is created for level up documentations.

**Comment by kisshetty** [comment]
Created for level up.

**Review by kisshetty — APPROVED** [review]

File: levelup-oracle-on-iaas/lab1/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Lab 1: Capacity Planning
```
Comment (kisshetty) [inline levelup-oracle-on-iaas/lab1/readme.md:1]: Each Module, we created the respective branch.

**Review by kisshetty — APPROVED** [review]
This is created for level up.

**Review by kisshetty — APPROVED** [review]

**Review by kisshetty — APPROVED** [review]

**Review by kisshetty — APPROVED** [review]

**Review by kisshetty — APPROVED** [review]
Approved.

**Review by jfaurskov — APPROVED** [review]
LGTM

**Pull request merged**